app/components/comments/comments.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="comments-container" [ngClass]="{'addition-padding': !isFooterVisible && userId, 'small-addition-padding': !userId}">
+  <div *ngFor="let comment of comments" class="comment-container">
+    <div class="comment-picture-container" [ngClass]="{'float-left': comment.owner.id != userId, 'float-right': comment.owner.id == userId}">
+      <img class="rounded-image" *ngIf="comment.owner.company && comment.owner.company.id" [src]="(comment.owner.company.companyLogo && comment.owner.company.imageUrlSmall) || 'assets/images/placeholder-logo.png'">
+      <img class="rounded-image" *ngIf="!(comment.owner.company && comment.owner.company.id)" [src]="(comment.owner.images && comment.owner.images[0] && comment.owner.images[0].imageUrl) || 'assets/images/placeholder-profile-image.png'">
```

Comment:
@TeodorTunev 

`comment.owner.images[0]` hardcoding the image like this can break this feature. Each user (and therefore owner) can have multiple images. Each image has a `category`, which identifies what kind of image it is. The available user categories are `profile` and `swedish_id`.

Also it seems like

```js
comment.owner.images && comment.owner.images[0] && comment.owner.images[0].imageUrl)
```
could be extracted  to its own function or property `comment.ownerProfileImage` or something like that :)